src/main/java/uk/gov/hmcts/darts/arm/mapper/impl/TranscriptionArchiveRecordMapperImpl.java

```diff
@@ -228,6 +233,7 @@ private RecordMetadata createArchiveRecordMetadata(ExternalObjectDirectoryEntity
     private String mapToString(String key, TranscriptionDocumentEntity transcriptionDocument) {
```

Comment:
It feels like we're missing `START_DATE_TIME` and END_DATE_TIME` in `transcription-record.properties` if you have these on lines 246 and 247?

Reply:
The start and end dates have been coded should these be required in the future, however they are currently not required.

Reply:
I feel like it would be less confusing if they were removed to be consistent with the same arguments around Annotation?